inst/include/cpp11/raws.hpp

```diff
@@ -52,11 +52,16 @@ inline typename r_vector<uint8_t>::underlying_type* r_vector<uint8_t>::get_p(
 }
 
 template <>
-inline void r_vector<uint8_t>::const_iterator::fill_buf(R_xlen_t pos) {
-  using namespace cpp11::literals;
-  length_ = std::min(64_xl, data_->size() - pos);
-  unwind_protect([&] { RAW_GET_REGION(data_->data_, pos, length_, buf_.data()); });
-  block_start_ = pos;
+inline void r_vector<uint8_t>::get_region(
+    SEXP x, R_xlen_t i, R_xlen_t n,
+    typename traits::get_underlying_type<uint8_t>::type* buf) {
+  // NOPROTECT: likely too costly to unwind protect here
+  RAW_GET_REGION(x, i, n, buf);
```

Comment:
No other implementation was using `unwind_protect()`, so I'm no longer doing it here either